app/frontend/src/features/communities/discussions/DiscussionPage.test.tsx

```diff
@@ -409,7 +409,7 @@ describe("Discussion page", () => {
         within(commentFormContainer).getByRole("button", { name: COMMENT })
       );
       // Check refetch loading state is shown while user is waiting for reply
-      expect(await screen.findByTestId(REFETCH_LOADING_TEST_ID)).toBeVisible();
+      await screen.findByTestId(REFETCH_LOADING_TEST_ID);
```

Comment:
It keeps failing on my machine. `findByTestId` will throw an error already if the element is not found. Which seemed to be already enough for me.

Reply:
Hmm it's a bit odd this would make a difference though. Although since this just loading state, it's more something we'd wait on rather than caring as the final result, so happy to leave it for this if this strangely makes a difference...